### ISSUE TITLE
Training PRN model

### ISSUE THREAD

**Issue by rafaelbou** [YadiraF/PRNet#93]
Hi everyone,
I began to train the model and I have some questions:

1. Dataset: I downloaded  300W_LP dataset and it contains ~200K images while the author report about ~60K images at the same dataset. Someone know what is the difference between these versions?

2. Overfitting: While training the dataset I'm getting in to overfit:
![image](https://user-images.githubusercontent.com/34336932/49931760-3fb0bc80-fed0-11e8-9e04-f63aa22f2a66.png)
someone manage same issue too?

3. About the loss function: Do you add L2 regularization to the loss function? Do you use the uv_weight_mask.png as it is? Do you multiply it with uv_face_mask.png to ignore the neck region? Do you use L1 or L2 for the loss calculation (uv' - uv)?

4. Evaluation: How do you evaluate your model? meanwhile I'm using only the loss function and visualize the output uv maps.

Thanks to the answers,
Rafael.

_Originally posted by @rafaelbou in https://github.com/YadiraF/PRNet/issues/46#issuecomment-446916937_

**Comment by qiannvMao** [comment]
@rafaelbou  Hi,as a newer to this, I failed to achieve the training code, could you please provide training code to me? I hope I can get your help, thank you very much. My email: mqn20170910@163.com

**Comment by lxzkeyan** [comment]
@rafaelbou  Can you share your training code?Looking forward to your reply.Thank you again.Email address: lxz_nxu@163.com

**Comment by xingxingsuo33** [comment]
@rafaelbou Can you share your training code?Looking forward to your reply.Thank you again.Email address: 549012844@qq.com

**Comment by yanghao960502** [comment]
可以分享一下你的训练代码吗？期待你的回复。再次感谢。邮箱 1363627028@qq.com
